cli/src/movement_util.rs

```diff
@@ -159,39 +160,29 @@ fn choose_commit<'a>(
     Ok(&commits[choice.parse::<usize>().unwrap() - 1])
 }
 
-pub fn move_to_commit(
+pub(crate) fn move_to_commit(
     ui: &mut Ui,
     workspace_command: &mut WorkspaceCommandHelper,
     direction: &Direction,
-    edit: bool,
-    has_conflict: bool,
-    change_offset: u64,
+    args: &mut MovementArgs,
```

Comment:
nit: It's weird that this function takes input parameter as `&mut`. Let's clone `args` in this function instead. This `.clone()` will be removed if we drop support for the "auto" behavior.

Reply:
Ah thanks. This is better. Done.